docs/APIReference-Renderer.md

```diff
@@ -0,0 +1,187 @@
+
+## Overview
+
+*Props*
+
+<ul class="apiIndex">
+  <li>
+    <a href="#container">
+      <pre>Container</pre>
+      Relay container that defines fragments and the view to render.
+    </a>
+  </li>
+  <li>
+    <a href="#forcefetch">
+      <pre>forceFetch</pre>
+      Whether to send a server request regardless of data available on the client.
+    </a>
+  </li>
+  <li>
+    <a href="#onforcefetch">
+      <pre>onForceFetch</pre>
+      Called whenever forceFetch is being called
+    </a>
+  </li>
+  <li>
+    <a href="#onprimecache">
+      <pre>onPrimeCache</pre>
+      Called whenever primeCache is being called
+    </a>
+  </li>
+  <li>
+    <a href="#queryconfig">
+      <pre>queryConfig</pre>
+       QueryConfig or Route that defines the query roots.
+    </a>
+  </li>
+  <li>
+    <a href="#environment">
+      <pre>environment</pre>
+      The API for Relay core that provides methods for fetching/mutating data, an in-memory cache, etc.
+    </a>
+  </li>
+    <li>
+    <a href="#render">
+      <pre>render</pre>
+      Called to render when data requirements are being fulfilled.
+    </a>
+  </li>
+  <li>
+    <a href="#onreadystatechange">
+      <pre>onReadyStateChange</pre>
+    </a>
+  </li>
+</ul>
+
+## Props
+
+### Container
+
+```
+Container: RelayContainer
+```
+
+Must be a valid `RelayContainer`. Relay will attempt to fulfill its data requirements before rendering it.
+
+
+### forceFetch
+
+```
+forceFetch: boolean
+```
+
+If supplied and set to true, a request for data will always be made to the server regardless of whether data on the client is available to immediately fulfill the data requirements.
+
+
+### onForceFetch
+
+```
+onForceFetch(
+  querySet: RelayQuerySet,
+  callback: (readyState: ReadyState) => void
+): Abortable
+```
+
+This callback prop is called whenever a forceFetch occurs.
+
+
+### onPrimeCache
+
+```
+onPrimeCache(
+  querySet: RelayQuerySet,
+  callback: (readyState: ReadyState) => void
+): Abortable
+```
+
+This callback prop is called whenever a primeCache call occurs.
+
+### QueryConfig
+
+```
+queryConfig: RelayRoute
+```
+
+Either an instance of `Relay.Route` or an object with the `name`, `queries`, and optionally the `params` properties.
+
+
+### Environment
+
+```
+environment: RelayEnvironment
+```
+
+Either `Relay.Store`, an instance of `Relay.Evironment` or an object with the `forceFetch`, `getFragmentResolver`, `getStoreData`, `primeCache`, and `read` function defined.
+
+
+### render
+
+```
+render(
+  props: ?{[propName: string]: mixed},
+  done: boolean,
+  error: ?Error,
+  retry: ?Function,
+  stale: boolean
+): ?RelayRenderCallback
+```
+
+Whenever anything is rendered `render` is called to render the view. If the callback is not supplied, the default behavior is to render the container if data is available, the existing view if one exists, or nothing.
```

Comment:
> Whenever anything is rendered `render` is called to render the view.

This sentence seems redundant to me. The word "render" is repeated three times, and I could reasonably expect this to be the behavior just based on the name, `render`.